README.adoc

```diff
@@ -551,12 +547,19 @@ You will see the following output:
 
 [source, role="no_copy"]
 ----
-60
+20
 ----
 
-The above command returns a value of 60, indicating that a total of 60 requests are made to the `system` service.
-The 3 default Istio requests for the `system` service, the 5 requests for the `inventory` service
-that you enabled in the [hotspot file=0]`traffic.yaml` file, and the 4 requests sent by MicroProfile are multiplied together.
+InventoryResource.java
+[source, Java, linenums, role='code_column hide_tags=copyright,fallback,fallbackMethod']
+----
+include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryResource.java[]
+----
+
+The above command returns a value of 20, indicating that a total of 20 requests are made to the `system` service that are multiplied by the following:
```

Comment:
Im not sure what is meant by " a total of 20 requests are made to the `system` service that are multiplied by the following:"

does it mean 5 Istio requests x 4 MP requests = 20?
or 20 x 4 x 5 = 400?

Reply:
it means -> 5 Istio requests x 4 MP requests = 20